### ISSUE TITLE
how to train the baseline model

### ISSUE THREAD

**Issue by zhangxueyangjuxie** [c4dm/dcase-few-shot-bioacoustic#15]
hi
  I try to train the baseline model use 2022 dataset, but the result is not well use the default setting, can you help me?

**Comment by shubhr** [comment]
Sure. What is the result that you are getting ?

**Comment by zhangxueyangjuxie** [comment]
I try to use default setting, get feature、train model use dataset2022 and eval my model，the F score near zero，but I test your model when using the same process，the F score is  26%，It seems to something wrong  with the training process，but I use the latest code and default setting, Can you help me check it.

================================
set:
  features: true
  train: true
  eval: true

path:
  root_dir:  /*/Bird_Dev_train
  train_dir:/*/DCASE_FSBioSED_task_22/Development_Set/Training_Set
  eval_dir:  /*/DCASE_FSBioSED_task_22/Development_Set/Validation_Set
  feat_path: ${path.root_dir}/Features
  feat_train: ${path.feat_path}/feat_train
  feat_eval:  ${path.feat_path}/feat_eval
  Model: ${path.root_dir}/Model
  best_model: ${path.Model}/best_model.pth
  last_model: ${path.Model}/last_model.pth

features:
  seg_len: 0.200
  hop_seg: 0.100
  eps: 2.220446049250313e-16
  fmax: 11025
  fmin: 50
  sr: 22050
  n_fft: 1024
  n_mels: 128
  hop_mel: 256
  
train:
  num_classes: 15
  n_shot: 5
  k_way : 5
  device: cuda
  lr_rate: 0.001
  scheduler_gamma: 0.5
  scheduler_step_size: 10
  epochs: 50
  num_episodes: 2000
  encoder: Resnet  # Protonet or Resnet
eval:
  samples_neg: 50
  iterations: 10
  query_batch_size: 8
  negative_set_batch_size : 16
  threshold: 0.45
===============================

**Comment by shubhr** [comment]
I am getting the baseline results with the default settings. Just running some tests on the training code. Will update you by today on the final outcome.

**Comment by haoheliu** [comment]
Me too. I cannot reproduce the baseline. The training and validation accuracy is always around 0.12.

**Comment by zhangxueyangjuxie** [comment]
![image](https://user-images.githubusercontent.com/15180528/160955443-aec3cd39-2977-4e1c-a03a-c6960a181fbb.png)
